corehq/apps/style/templates/style/crispy/multi_inline_field.html

```diff
@@ -0,0 +1,25 @@
+{% load crispy_forms_field %}
+{% load i18n %}
+
+{% if field.is_hidden %}
+    {{ field }}
+    {% if field.errors %}
+        <div class="alert alert-danger">
```

Comment:
we use crispy with B2 and B3, but  this specific field should only ever be used with B3 because it's only meant to be referenced within a B3MultiField